chat-plugins/info.js

```diff
@@ -426,8 +426,8 @@ exports.commands = {
 					}
 					return this.sendReply(buffer);
 				case 'pokemon':
-					let template = Object.assign({}, mod.getTemplate(newTarget.name));
-					if ((room && room.id === 'smogondoubles') || (room && ['gen7doublesou', 'gen7doublesubers', 'gen7doublesuu'].includes(toId(Dex.getFormat(room.battle.name))))) template.tier = template.doublesTier;
+					let template = mod.getTemplate(newTarget.name);
+					if (room.id === 'smogondoubles' || ['gen7doublesou', 'gen7doublesubers', 'gen7doublesuu'].includes(Dex.getFormat(room.battle).id)) template.tier = template.doublesTier;
```

Comment:
no :|

Reply:
just pass an override tier to `getDataPokemonHTML`

Reply:
im confused by what you mean

Reply:
You're overwriting `template`, which is a big no-no

Reply:
I understand that but how would I handle it in getDataPokemonHTML I’ve tried before

Reply:
You could just, not do it and leave it to me, I guess.

Reply:
Actually I'll just show you the code:

```
Chat.getDataPokemonHTML = function (template, gen = 7, tierString = '') {
```

...

```
	buf += '<span class="col numcol">' + (tierString || template.tier) + '</span> ';
```

Reply:
I didn't notice that you'd pushed this already so I've been spending the past hour working on this and wondering why my PR wasn't updating